kahuna/public/js/components/gr-downloader/gr-downloader.html

```diff
@@ -3,19 +3,29 @@
     <button type="button"
             title="Download images"
             ng-disabled="ctrl.downloading"
-            ng-if="ctrl.imageCount() > 1"
+            ng-if="ctrl.imageCount() > 1 && ctrl.canDownload"
             ng-click="ctrl.download('downloadUri')">
         <gr-icon-label gr-icon="file_download">
             Download<span ng-show="ctrl.downloading">ing</span>
         </gr-icon-label>
     </button>
 
-    <a ng-if="ctrl.imageCount() == 1"
+    <span ng-if="ctrl.imageCount() > 1 && !ctrl.canDownload" aria-label="Cannot download" gr-tooltip="Cannot download the selected images.">
```

Comment:
@paperboyo Any suggestions on what to put for `gr-tooltip` messaging? I left it quite generic for now but happy to follow your suggestion

Reply:
1. Is this for multiple selected images? If yes – does one non-downloadable one block the whole download (even if others are fine)? I think it shouldn’t. The download zip should either just silently nopt have non-downloadable images (bad) or ditto, but with some kind of messaging (also bad, no idea how to message without a blocking popup).
2. No idea what would be the best tooltip. Maybe just “Downloading requires a lease” for any Cannot download button (where either one or all images are non-downloadable)?
    ![image](https://user-images.githubusercontent.com/6032869/182119050-490994b4-d348-47d3-b5ca-a7dd9a989d1c.png)

Reply:
Yes for now a single non-downloadable image would block the whole download. As you said, not sure where messaging should be positioned in that case, and silently ignoring it could maybe annoy the users? Not sure. 

Thanks for the suggestion on the tooltip, will update that.

Reply:
Definitely let’s not block the whole batch download. Maybe we can repurpose some banner code to non-blockingly say: x images won’t download because they need lease(s) after clicking Download?

Reply:
Is there an example banner that's currently in use that's similar to what you have in mind? No worries if not, I can have a look in the code as well

Reply:
There is one on crop page when crop is too small (non-blocking). There is also a cookie one (blocking I think?).